### ISSUE TITLE
Impossible to set environment variables for debuggee

### ISSUE THREAD

**Issue by jalfd** [microsoft/WinDbg-Feedback#208]
Perhaps I'm just missing something obvious but there doesn't seem to be a good/easy/reliable way to run a process under windbgx with a specific environment set.

It appears that environment variables I set in the terminal before launching windbx do not actually get carried forward to the debuggee when I launch that from within Windbg. Perhaps this is because windbgx is a store application with its own weird behavior?

There's also no UI under "Launch Executable" for specifying environment variables.

This feels like such a common use case that I'm wondering if I'm missing a trick. In either case, not exposing this functionality in the UI feels like an omission

**Comment by lzybkr** [comment]
Agreed, inheriting environment variables should work, but we should also have UI to set them when launching.

**Comment by psionic12** [comment]
So, anyone knows how to work around this? I really need set ENVs when launching a program.

**Comment by lzybkr** [comment]
As @jalfd mentions - environment variables can be set in the console:

In cmd:
```cmd
set MY_ENV_VAR=something
windbgx my_process.exe
set MY_ENV_VAR=
```

Or PowerShell:

```powershell
$env:MY_ENV_VAR = "something"
windbgx my_process.exe
Remove-Item env:MY_ENV_VAR
```

**Comment by KenSykes** [comment]
I would do what @jalfd mentions but there is another option too: make a .cmd file that sets the environment variables and calls the target program. Then, use "cmd /c my.cmd" as the command line, with "debug child processes" enabled.

**Comment by R-Goc** [comment]
Ran into this today. Maybe this could be a part of launch executable (advanced). Though a full environment pane to edit freely would be nice of course.
